src/main/java/com/csemosip/bookingservice/service/impl/AuthenticationServiceImpl.java

```diff
@@ -32,22 +29,26 @@ public AuthenticationResponse login(AuthDTO authDTO) {
             // Call the external authentication endpoint
             // Once authenticated externally, can proceed to generate JWT token
 
-//            authenticationManager.authenticate(
-//                    new UsernamePasswordAuthenticationToken(
-//                            authDTO.getUsername(),
-//                            authDTO.getPassword()
-//                    )
-//            );
+            String username = authDTO.getUsername();
+            String password = authDTO.getPassword();
+            RestTemplate restTemplate = new RestTemplate();
+            //TODO: Replace with the correct endpoint
+            String verificationEndPoint = "https://36811170-f4c5-404c-90ad-3d807b36f503.mock.pstmn.io/api/public/verify";
```

Comment:
I think it would be better to get this endpoint from application.properties file.